scripts/net-traffic

```diff
@@ -143,44 +154,6 @@ RX=$(echo "scale = 0; ($RX2-$RX1)/$DLY" | bc -l)
 TX=$(echo "scale = 0; ($TX2-$TX1)/$DLY" | bc -l)
 AX=$(echo "scale = 0; $RX+$TX" | bc -l)
 
-# set net unit automatically
```

Comment:
This can be removed because it's redundant?

Reply:
Yes, I’m getting rid of it in favor of a sane default (which also fixes a shellcheck error)

Reply:
We could keep both around though, if you prefer to have the code instead of the (arguable, forced) default